system-addon/test/unit/content-src/components/Search.test.jsx

```diff
@@ -50,7 +50,7 @@ describe("<Search>", () => {
 
     wrapper.find(".search-button").simulate("click");
 
-    const {search} = wrapper.node.controller;
+    const {search} = window.gContentSearchController;
```

Comment:
I suppose this effectively tests that we place it on the global. Might make sense to have a test dedicated to just making sure `window.gContentSearchController` exists after mounting with a description of it being an expected global.